rijkshuisstijl/sass/components/layout/_layout-small.scss

```diff
@@ -0,0 +1,10 @@
+@import '~microscope-sass/lib/grid';
+@import 'mixins';
+
+.layout {
+  @include layout;
```

Comment:
Ik denk dat je deze het beste shared kan maken tussen alle layout types. En puur de modifier logica in de losse bestanden stoppen.

Reply:
Ik heb ze nu in drieen gesplitst, met de aanname dat het auto-column gebeuren bij de _layout-1col.scss horen